ocaml/testsuite/tests/parsetree/locations_test.compilers.reference

```diff
@@ -243,15 +245,22 @@ Ptop_def
             Ppat_constraint
             pattern (//toplevel//[2,1+4]..[2,1+5])
               Ppat_var "x" (//toplevel//[2,1+4]..[2,1+5])
-            core_type (//toplevel//[2,1+16]..[2,1+22]) ghost
-              Ptyp_poly 'a
-              core_type (//toplevel//[2,1+16]..[2,1+22])
-                Ptyp_arrow
-                Nolabel
-                core_type (//toplevel//[2,1+16]..[2,1+17])
-                  Ptyp_var a
-                core_type (//toplevel//[2,1+21]..[2,1+22])
-                  Ptyp_var a
+            Some
```

Comment:
I wonder if we should just not print this `Some` - it doesn't seem to give more information.